scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/scripts/clone-kibana.sh

```diff
@@ -1,9 +1,12 @@
+#!/usr/bin/env bash
+set -e
+
 OWNER=${1:-elastic}
 BRANCH=${2:-master}
 
-echo "Cloning Kibana: $OWNER:$BRANCH"
+echo "Cloning Kibana: ${OWNER}:${BRANCH}"
 
-cd ./tmp
-git clone --depth 1 -b $BRANCH https://github.com/$OWNER/kibana.git
-mv ./kibana/x-pack/legacy/plugins/apm/typings/es_schemas ./apm-ui-interfaces
+cd ./tmp || true
```

Comment:
Fix a lint defect

Reply:
What does this do? Is it if it fails navigating to `./tmp`?

Reply:
Yes, have you got a different suggestion?

Reply:
I'm wondering why this would fail - and fix the root cause of that.

Reply:
I'll keep as it was then

Reply:
rollback my changes